moveit_ros/trajectory_cache/CHANGELOG.rst

```diff
@@ -0,0 +1,7 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package nexus_motion_planner
```

Comment:
Update this package name

Reply:
85aa3f792825af979f4b3a13ac14391b0746fc59